tools/bin/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+svn_url=https://svn.apache.org/repos/asf/cordova/site
+
+svn_q() { svn --quiet "$@" }
+
+svn_with_auth() {
+    [ $svn_username ] && [ $svn_pass ] || {
+        echo 'ERROR: No SVN credentials given in $svn_username and $svn_pass'
+        exit 1
+    }
+    svn_q --non-interactive --no-auth-cache \
+          --username="$svn_username" --password="$svn_pass" "$@"
+}
+
+echo "Deploying website and docs to $svn_url"
+cd ..
+svn_q checkout $svn_url cordova-website
```

Comment:
Please put the URL in quotes.

Reply:
I thought it will probably never contain spaces, but I know myself that that's a weak argument. Will do.

Reply:
Yes. Also see the CI run for this PR.